sorting_algos/quicksort/quicksort.py

```diff
@@ -0,0 +1,19 @@
+def quicksort(lst):
+    """Sort a list of integers."""
+    if type(lst) is not list:
+        raise TypeError('argument must be type <list>')
+    if not len(lst):
```

Comment:
When checking if a list is empty, you do not need to check the `len`. An empty list is falsey, so you can just check `if not lst`.